bin/extract_api_surface.py

```
@@ -0,0 +1,207 @@
+            s = kw_arg.arg
+            ann = _annotation_to_str(kw_arg.annotation) if kw_arg.annotation else None
+            if ann:
+                s += f": {ann}"
+            params.append(s)
+    if func_node.args.kwarg:
+        params.append(f"**{func_node.args.kwarg.arg}")
+
+    return f"({', '.join(params)})"
+
+
+def _extract_class_methods(tree, class_name):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_extract_class_methods`

(ANN202)